test/e2e/node/terminal.go

```diff
@@ -0,0 +1,86 @@
+package node
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+
+	"github.com/gorilla/websocket"
+	"github.com/onsi/ginkgo"
+
+	v1 "github.com/kubeclipper/kubeclipper/pkg/apis/core/v1"
+	"github.com/kubeclipper/kubeclipper/pkg/authentication/oauth"
+	"github.com/kubeclipper/kubeclipper/pkg/query"
+	"github.com/kubeclipper/kubeclipper/pkg/scheme/common"
+	"github.com/kubeclipper/kubeclipper/pkg/simple/client/kc"
+	"github.com/kubeclipper/kubeclipper/pkg/utils/certs"
+	"github.com/kubeclipper/kubeclipper/test/framework"
+)
+
+var _ = SIGDescribe("[Slow] [Serial] Node terminal connect", func() {
+	f := framework.NewDefaultFramework("node")
+	msg := ""
+	nodeID := ""
+	token := &oauth.Token{}
+	credential := &v1.SSHCredential{}
+
+	ginkgo.BeforeEach(func() {
+		ginkgo.By("Check that there are enough available nodes")
+		nodes, err := f.Client.ListNodes(context.TODO(), kc.Queries{
+			Pagination:    query.NoPagination(),
+			LabelSelector: fmt.Sprintf("!%s", common.LabelNodeRole),
+		})
+		framework.ExpectNoError(err)
+		nodeID = nodes.Items[0].Name
+
+		ginkgo.By("Get public key")
+		pub, err := f.Client.GetPublicKey(context.TODO())
+		framework.ExpectNoError(err)
+
+		ginkgo.By("Get token")
+		body := kc.LoginRequest{
+			Username: "admin",
+			Password: "Thinkbig1",
+		}
+		token, err = f.Client.Login(context.TODO(), body)
```

Comment:
kc.client add a  method to get token,like get host,refer pkg/simple/client/kc/client.go